cardano-tx-submit/Readme.md

```diff
@@ -0,0 +1,34 @@
+# cardano-tx-submit-webapi
+
+The `cardano-tx-submit-webapi` is a program that provides a webapi that allows transactions
+(generated using the `cardano-cli` program in the `cardano-node` repository) to be HTTP POSTed
+to the Cardano blockchain.
+
+In order to submit a transaction to the network (mainnet, staging or any of the testnets), the
+webapi needs a running `cardano-node` and the Genesis file and Genesis hash value for the network.
+
+The config for `mainnet` is provided at `config/tx-submit-mainnet-config.yaml` of this repository.
+
+Configuration for other networks can be generated using the above as a template using:
+```
+scripts/generate-tx-submit-config.sh --require-magic --genesis-hash <hash> --output config.yaml
```

Comment:
Just a typo: The actual name of the script is: `gen-tx-submit-config.sh` not `generate-tx-submit-config.sh`